securedrop_client/gui/login_dialog.py

```diff
@@ -35,12 +35,8 @@
 )
 
 from securedrop_client import __version__ as sd_version
-from securedrop_client.gui.widgets import (
-    LoginErrorBar,
-    LoginOfflineLink,
-    PasswordEdit,
-    SignInButton,
-)
+from securedrop_client.gui.auth import SignInButton
+from securedrop_client.gui.widgets import LoginErrorBar, LoginOfflineLink, PasswordEdit
```

Comment:
```diff
- from securedrop_client.gui.widgets import LoginErrorBar, LoginOfflineLink, PasswordEdit
+ from securedrop_client.gui.base import LoginErrorBar, LoginOfflineLink, PasswordEdit
```

Will become #1378 is merged. (That's the only change that the rebase will bring.